### PR TITLE
[NOJIRA] New import/order rule in eslint

### PR DESCRIPTION
**Pull request by matthewdavidson**
Related PR: ~https://github.com/Skyscanner/eslint-config-skyscanner/pull/21~ https://github.com/Skyscanner/eslint-config-skyscanner-with-prettier/pull/19

### CODE REVIEW

**Comment by matthewdavidson** [comment]
Depends on https://github.com/Skyscanner/eslint-config-skyscanner-with-prettier/pull/19

File: packages/bpk-docs/src/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -46,6 +40,9 @@ import LondonHeroImage from '../../static/london_hero.jpg';
 import HongKongHeroImage from '../../static/hongKong_hero.jpg';
 import DohaHeroImage from '../../static/doha_hero.jpg';
 
+import UpdatedAt from './UpdatedAt';
+import STYLES from './home-page.scss';
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/pages/HomePage/HomePage.js]: FYI @sogame, this change broke some layout in the backpack homepage 😅 (see the fix here https://github.com/Skyscanner/backpack/pull/1322).

It's a bit of a gotcha that I wan't to add to the readme of `eslint-config-skyscanner` - however I reckon it actually highlights some bad practices in terms of component concerns anyway.

Comment (sogame) [reply packages/bpk-docs/src/pages/HomePage/HomePage.js]: Oh! Sorry about that @matthewdavidson 😓
Good point about adding a note in the readme (and maybe also in the changelog, as users upgrading might only check the changelog?)

Silver lining for me: even Backpack do this kind of mistakes 😝

Comment (matthewdavidson) [reply packages/bpk-docs/src/pages/HomePage/HomePage.js]: It's not your fault!! 😂 Just a heads up that this kind of thing _can_ happen.

**Pull request merged**